quixstreams/sources/community/file/file.py

```diff
@@ -63,7 +79,8 @@ class FileSource(Source):
     def __init__(
         self,
         filepath: Union[str, Path],
-        file_format: Union[Format, FormatName],
+        file_format: Union[Format, FormatName] = "json",
+        file_origin: Origin = LocalOrigin(),
         file_compression: Optional[CompressionName] = None,
```

Comment:
Hey, a suggestion here: since we're changing the API, let's keep the names similar to `FileSink`.

- `filepath` -> `directory`
- `file_format` -> `format`
- `file_origin` -> `origin`
- `file_compression` -> `compression`